app/views/articles/_widget_list_item.html.erb

```diff
@@ -2,8 +2,10 @@
   <%= plucked_article[1] %>
   <% if show_comment_count %>
     <div class="crayons-link__secondary">
-      <% if plucked_article[2] > 0 %>
-        Comments: <%= plucked_article[2] %>
+      <% if plucked_article[2] == 1 %>
+        <%= plucked_article[2] %> comment
+      <% elsif plucked_article[2] > 1 %>
+        <%= plucked_article[2] %> comments
```

Comment:
you could use this Rails trick: 

```ruby
<%= pluralize(plucked_article[2], 'comment') %>
```

which automatically decides if to write `comment` or `comments` if the number is above 1. 

see https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize

Reply:
@lisasy I'd definitely say making this adjustment would be worth it for code cleanliness. Change overall looks good though.

Reply:
@rhymes thank you, this is so helpful 😁